projects/packages/identity-crisis/src/class-identity-crisis.php

```diff
@@ -1394,4 +1395,65 @@ public static function register_request_body( array $params ) {
 	public static function site_registered( $blog_id ) {
 		update_option( static::PERSISTENT_BLOG_ID_OPTION_NAME, (int) $blog_id, false );
 	}
+
+	/**
+	 * If URL is an IP, add the IP value to the ip_requester option with its expiry value.
+	 *
+	 * @throws Exception Thrown if unable to save the new IP requester.
+	 */
+	public static function set_ip_requester_for_idc() {
```

Comment:
Should we add some sort of transient here?
I think we could skip running the code below if it's already been run for that `siteurl` in the last few (five?) minutes.

Reply:
Good idea. Transient added in 3567c969ed36f5b49a5f220776ba85fea3b7ea7b